queryfilter/utils.py

```diff
@@ -0,0 +1,10 @@
+def is_same_class(x_cls, y_cls):
```

Comment:
Not so sure that it will be used in other place, maybe put it back to the file where it's used for now?